pds/contracts/PDS.cdc

```diff
@@ -1,38 +1,40 @@
-import NonFungibleToken from 0x{{.NonFungibleToken}}
-import IPackNFT from 0x{{.IPackNFT}}
+import NonFungibleToken from "NonFungibleToken"
+import IPackNFT from "IPackNFT"
+
+access(all) contract PDS{
+    /// Entitlement that grants the ability to create a distribution
+    access(all) entitlement DistCreation
```

Comment:
New entitlement to control access for distribution creation, replacing dist creator resource interface in pre-Cadence 1.0 contract